### PR TITLE
OCPBUGS-17950: Use OLMconfig to specify package server interval

### PR DESCRIPTION
**Pull request by tmshort**

### CODE REVIEW

File: staging/operator-lifecycle-manager/pkg/package-server/server/server.go [hunk 1]
```diff
@@ -16,14 +16,29 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/workqueue"
 
+	operatorsv1 "github.com/operator-framework/api/pkg/operators/v1"
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/informers/externalversions"
```
Comment (stevekuznetsov) [inline staging/operator-lifecycle-manager/pkg/package-server/server/server.go:24]: nit: it's nice to alias this to `olminformers` or something similar, `externalversions` is a nondescript name

File: staging/operator-lifecycle-manager/pkg/package-server/server/server.go [hunk 2]
```diff
@@ -16,14 +16,29 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/workqueue"
 
+	operatorsv1 "github.com/operator-framework/api/pkg/operators/v1"
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
```
Comment (stevekuznetsov) [inline staging/operator-lifecycle-manager/pkg/package-server/server/server.go:23]: nit: it's nice to alias this to `olmclientset` or something similar, `versioned` is a nondescript name

File: staging/operator-lifecycle-manager/pkg/package-server/server/server.go (hunk 1 above)
Comment (tmshort) [reply staging/operator-lifecycle-manager/pkg/package-server/server/server.go:26]: I believe I took this from elsewhere, but I'm ok with aliasing.

**Comment by tmshort** [comment]
Committing upstream first.

**Comment by tmshort** [comment]
Everything passed, closing again.

**Pull request closed without merging**